src/app/settings/action-triggers/components/action-triggers-list/action-triggers-list.component.ts

```diff
@@ -0,0 +1,226 @@
+  @Output() private readonly onActionTriggerOpen: EventEmitter<ActionTriggerWithAction<KeyboardTriggerData>> = new EventEmitter<ActionTriggerWithAction<KeyboardTriggerData>>()
+  public search: string = ''
+  public sortQuery: string = `${ActionTriggerSortKeys.ACTION}_${SortOrder.ALPHABETICAL}`
+  public readonly sortLabel: string = $localize`global.sort.label`
+  public readonly IconButton = IconButton
+  public readonly IconButtonSize = IconButtonSize
+  public readonly sortActionsTriggers: SofieDropdownOption[] = [
+    { key: `${ActionTriggerSortKeys.ACTION}_${SortOrder.ALPHABETICAL}`, label: $localize`action-triggers-sort.action-label-alphabetical.label` },
+    { key: `${ActionTriggerSortKeys.ACTION}_${SortOrder.REVERSE_ALPHABETICAL}`, label: $localize`action-triggers-sort.action-label-reverse-alphabetical.label` },
+    { key: `${ActionTriggerSortKeys.SHORTCUT}_${SortOrder.ALPHABETICAL}`, label: $localize`action-triggers-sort.shortcut-alphabetical.label` },
+    { key: `${ActionTriggerSortKeys.SHORTCUT}_${SortOrder.REVERSE_ALPHABETICAL}`, label: $localize`action-triggers-sort.shortcut-reverse-alphabetical.label` },
+  ]
+
+  public selectedActionTriggerOptions: SofieDropdownOption[] = [
+    { key: ActionsWithSelected.TOGGLE_SELECT, label: $localize`global.select-all.label`, isDisabled: false },
+    { key: ActionsWithSelected.EXPORT, label: $localize`global.export-selected.label`, isDisabled: true },
+    { key: ActionsWithSelected.DELETE, label: $localize`global.delete-selected.label`, isDisabled: true },
+  ]
+  public selectMode: boolean = false
+  private readonly selectedActionTriggerIds: Set<string> = new Set()
+
+  constructor(
+    private readonly dialogService: DialogService,
+    private readonly fileDownloadService: FileDownloadService,
+    private readonly actionTriggerService: ActionTriggerService
+  ) {}
+
+  public ngOnChanges(changes: SimpleChanges): void {
+    const actionTriggerChange: SimpleChange | undefined = changes['actionTriggers']
+    if (actionTriggerChange) {
+      this.newSortSelect(this.sortQuery)
+    }
+  }
+
+  get filteredActionsTriggers(): ActionTriggerWithAction<KeyboardTriggerData>[] {
+    const lowercasedSearchQuery: string = this.search.toLocaleLowerCase()
+    return this.actionTriggersWithAction.filter(
+      actionTriggerWithAction =>
+        actionTriggerWithAction.actionTrigger.data.label.toLocaleLowerCase().includes(lowercasedSearchQuery) ||
+        this.getMappetToOrShortcutKeysArray(actionTriggerWithAction.actionTrigger).toString().toLocaleLowerCase().includes(lowercasedSearchQuery) ||
+        actionTriggerWithAction.action.name.toString().toLocaleLowerCase().includes(lowercasedSearchQuery) ||
+        actionTriggerWithAction.actionTrigger.id === this.selectedActionTrigger?.actionTrigger.id
+    )
+  }
+
+  private getMappetToOrShortcutKeysArray(actionTrigger: ActionTrigger<KeyboardTriggerData>): Keys {
```

Comment:
`Array` is redundant and `Mappet` should be `Mapped`: `getMappedToOrShortcutKeys`

Reply:
Done